frappe/integrations/doctype/google_calendar/google_calendar.py

```diff
@@ -171,7 +172,12 @@ def get_google_calendar_object(g_calendar):
 	}
 
 	credentials = google.oauth2.credentials.Credentials(**credentials_dict)
-	google_calendar = googleapiclient.discovery.build("calendar", "v3", credentials=credentials)
+	google_calendar = build(
+		serviceName="calendar",
+		version="v3",
+		credentials=credentials,
+		static_discovery=False
```

Comment:
Can you comment on the reason for adding `static_discovery=False` throughout?

_(Perhaps through your commit message)_

Reply:
https://github.com/googleapis/google-api-python-client/blob/master/UPGRADING.md

With v2 and above, they've added an additional option for fetching resources - the client library.

By default, in versions <2, the resources would be fetched via the internet, whereas it defaults to being fetched from the client library in v2.

With the `static_discovery` being set to `False`, I wanted to maintain functionality and fetch from the internet always.